shop/src/main/java/ro/msg/learning/ShopApplication.java

```diff
@@ -0,0 +1,20 @@
+package ro.msg.learning;
+
+import java.io.FileNotFoundException;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@SpringBootApplication
+@EntityScan(basePackages = { "ro.msg.learning.entity" }) // scan JPA entities
+@EnableJpaRepositories(basePackages = "ro.msg.learning.repository") // scan JPA DAO's
```

Comment:
should work without these two annotations, now that you have the app class in the base package. packages will be scanned automatically

Reply:
Fixed